<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" ignored="0" total="4" passed="2">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="34641" started-at="2020-07-07T01:03:20 IST" finished-at="2020-07-07T01:03:55 IST">
    <groups>
    </groups>
    <test name="TestMotorola" duration-ms="24912" started-at="2020-07-07T01:03:20 IST" finished-at="2020-07-07T01:03:45 IST">
      <class name="framework.SearchTest">
        <test-method status="PASS" signature="TestSetUp(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:framework.SearchTest@e320068]" name="TestSetUp" is-config="true" duration-ms="21381" started-at="2020-07-07T01:03:20 IST" finished-at="2020-07-07T01:03:42 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[ZF6223T9M7]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Moto One Power]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[10]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TestSetUp -->
        <test-method status="FAIL" signature="sampleFailTestCase(java.lang.String)[pri:0, instance:framework.SearchTest@e320068]" name="sampleFailTestCase" duration-ms="31" started-at="2020-07-07T01:03:42 IST" finished-at="2020-07-07T01:03:42 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[Moto One Power]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [4] but found [2]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [4] but found [2]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:839)
at org.testng.Assert.assertEquals(Assert.java:849)
at framework.SearchTest.sampleFailTestCase(SearchTest.java:37)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1249)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:41)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:443)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:67)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
at java.lang.Thread.run(Thread.java:745)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- sampleFailTestCase -->
        <test-method status="PASS" signature="samplePassTestCase(java.lang.String)[pri:0, instance:framework.SearchTest@e320068]" name="samplePassTestCase" duration-ms="825" started-at="2020-07-07T01:03:42 IST" finished-at="2020-07-07T01:03:42 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[Moto One Power]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- samplePassTestCase -->
        <test-method status="PASS" signature="TearDown()[pri:0, instance:framework.SearchTest@e320068]" name="TearDown" is-config="true" duration-ms="2588" started-at="2020-07-07T01:03:42 IST" finished-at="2020-07-07T01:03:45 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TearDown -->
      </class> <!-- framework.SearchTest -->
    </test> <!-- TestMotorola -->
    <test name="TestGooglePixel" duration-ms="34641" started-at="2020-07-07T01:03:20 IST" finished-at="2020-07-07T01:03:55 IST">
      <class name="framework.SearchTest">
        <test-method status="PASS" signature="TestSetUp(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:framework.SearchTest@4d50efb8]" name="TestSetUp" is-config="true" duration-ms="29573" started-at="2020-07-07T01:03:20 IST" finished-at="2020-07-07T01:03:50 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[localhost:5555]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Google Pixel 2XL]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[7.1.2]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TestSetUp -->
        <test-method status="FAIL" signature="sampleFailTestCase(java.lang.String)[pri:0, instance:framework.SearchTest@4d50efb8]" name="sampleFailTestCase" duration-ms="6" started-at="2020-07-07T01:03:50 IST" finished-at="2020-07-07T01:03:50 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[Google Pixel 2XL]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [4] but found [2]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [4] but found [2]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:839)
at org.testng.Assert.assertEquals(Assert.java:849)
at framework.SearchTest.sampleFailTestCase(SearchTest.java:37)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1249)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:41)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:443)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:67)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
at java.lang.Thread.run(Thread.java:745)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- sampleFailTestCase -->
        <test-method status="PASS" signature="samplePassTestCase(java.lang.String)[pri:0, instance:framework.SearchTest@4d50efb8]" name="samplePassTestCase" duration-ms="51" started-at="2020-07-07T01:03:50 IST" finished-at="2020-07-07T01:03:50 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[Google Pixel 2XL]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- samplePassTestCase -->
        <test-method status="PASS" signature="TearDown()[pri:0, instance:framework.SearchTest@4d50efb8]" name="TearDown" is-config="true" duration-ms="4987" started-at="2020-07-07T01:03:50 IST" finished-at="2020-07-07T01:03:55 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TearDown -->
      </class> <!-- framework.SearchTest -->
    </test> <!-- TestGooglePixel -->
  </suite> <!-- Suite -->
</testng-results>
